src/Forms.js

```diff
@@ -118,13 +113,15 @@ export default function Forms() {
             />
           </div>
           <p>Choose a bank:</p>
-          <select value={bank} onChange={(event) => setBank(event.target.value)}>
-            {banks}
-          </select>
-
+          <Select value={bank}
+            displayEmpty
+            onChange={(event) => setBank(event.target.value)}>
+            <MenuItem value="" disabled>Select a bank</MenuItem>
+            <MenuItem value={"Sberbank"}>Sberbank</MenuItem>
+            <MenuItem value={"VTB Bank"}>VTB Bank</MenuItem>
+            <MenuItem value={"Alfa-bank"}>Alfa-bank</MenuItem>
```

Comment:
We shouldn't hardcode banks list. It still must be an iterator over the fixtures:
```
banks.map(bank => (<MenuItem value={bank}>{bank}</MenuItem>));
```

Reply:
fixed